src/neuroglancer/ui/graph_multicut.ts

```diff
@@ -398,6 +401,27 @@ export class GraphOperationLayerView extends Tab {
     this.multicutGroup.appendFlexibleChild(this.annotationListContainer);
     this.element.appendChild(this.multicutGroup.element);
 
+    const displayState =
+        <SegmentationUserLayerWithGraphDisplayState>(<GraphOperationLayerState>annotationLayer)
+            .segmentationState.value;
+
+    if (displayState && displayState.timestamp) {
+      this.timeWidget = this.registerDisposer(new TimeSegmentWidget(displayState, wrapper.manager.layerManager.messageWithUndo));
+      const disableToolbar = () => {
+        Array.from(toolbox.children).forEach((ele: HTMLButtonElement) => {
```

Comment:
Since we're only setting one element to disabled, it's easier to read if we select that element by class or id, instead of looping through the toolbox's children